TourGuide.lua

```diff
@@ -106,6 +108,18 @@ function TourGuide:Enable()
 	self:QuestID_QUEST_LOG_UPDATE()
 	GetQuestsCompleted(TourGuide.turnedinquests)
 	self:UpdateStatusFrame()
+	if IsLoggedIn() then
+		GetQuestsCompleted(TourGuide.turnedinquests)
+		self:UpdateStatusFrame()
+	else
+		self:RegisterEvent("PLAYER_LOGIN")
+		function self:PLAYER_LOGIN()
+			GetQuestsCompleted(TourGuide.turnedinquests)
+			self:UpdateStatusFrame()
+			self:UnregisterEvent("PLAYER_LOGIN")
+			self.PLAYER_LOGIN = nil
+		end
+	end
 end
```

Comment:
In the original version, the idea was to load quests when the user logs in. (But previously there was a prequery that had to be done before it was possible)

This kinda mimics that.

Reply:
I'm not sure, but I think we should follow the same base logic for now. That way we can avoid bugs which we might find out later on (those might be really hard to debug).